src/main/java/com/stripe/net/APIResource.java

```diff
@@ -61,6 +62,7 @@ public static void setStripeResponseGetter(StripeResponseGetter srg) {
 			.registerTypeAdapter(OrderItem.class, new OrderItemDeserializer())
 			.registerTypeAdapter(Source.class, new SourceTypeDataDeserializer<Source>())
 			.registerTypeAdapter(SourceTransaction.class, new SourceTypeDataDeserializer<SourceTransaction>())
+			.registerTypeAdapter(SourceMandateNotification.class, new SourceTypeDataDeserializer<SourceTransaction>())
```

Comment:
Second argument should be `new SourceTypeDataDeserializer< SourceMandateNotification>`.

Reply:
Also the list is in alphabetical order, so `SourceMandateNotification` should appear before `SourceTransaction`.

Reply:
@ob-stripe OOC why did this work then?